internal/work/dbus.go

```diff
@@ -22,9 +22,10 @@ func NewDBusError(name string, body ...string) *dbus.Error {
 func ScrubName(name string) (string, error) {
 	r := regexp.MustCompile("-")
 	if r.Match([]byte(name)) {
+		newName := r.ReplaceAll([]byte(name), []byte("_"))
 		return string(
-			r.ReplaceAll([]byte(name), []byte("_")),
-		), fmt.Errorf("invalid worker name: %v", name)
+			newName,
+		), fmt.Errorf("worker name: %v transformed to %v", name, newName)
```

Comment:
Given the rewritten error message, we should probably drop the colon:
```suggestion
		), fmt.Errorf("worker name %v transformed to %v", name, newName)
```